packages/ui/components/SuggestionOverview.tsx

```diff
@@ -24,22 +27,22 @@ function getCostlyPackages(
 
   suggestions.forEach((suggestion) => {
     suggestion.actions.forEach((action) => {
-      const parts = action.meta.breadcrumb.split('#');
+      const node = action.targetPackage;
 
-      if (parts.length) {
-        const associatedActionPackageName = parts[0];
-        if (!countMap[associatedActionPackageName]) {
-          console.log(`Missed: ${associatedActionPackageName}`);
-          const suggestionCount = {
-            packageName: associatedActionPackageName,
-            count: 1,
-          };
-          countMap[associatedActionPackageName] = suggestionCount;
-          counts.push(suggestionCount);
-        } else {
-          countMap[associatedActionPackageName].count++;
-        }
-      }
+      // if (parts.length) {
+      //   const associatedActionPackageName = parts[0];
+      //   if (!countMap[associatedActionPackageName]) {
+      //     console.log(`Missed: ${associatedActionPackageName}`);
+      //     const suggestionCount = {
+      //       packageName: associatedActionPackageName,
+      //       count: 1,
+      //     };
+      //     countMap[associatedActionPackageName] = suggestionCount;
+      //     counts.push(suggestionCount);
+      //   } else {
+      //     countMap[associatedActionPackageName].count++;
+      //   }
+      // }
     });
```

Comment:
is this meant to be commented out? Seems like the counts won't work if we dont have some sort of log here, maybe there is something further along in the PR that im missing

Reply:
Cool, sounds good